website/docs/foundations/colors/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,99 @@
+
+HDS organizes semantic tokens into four categories:
+
+- **Foreground** - For elements such as text, links, statuses, and icons.
+- **Border** - For borders on components, containers or dividers.
+- **Surface** - For the background (or surface) of a component or container
+- **Page** - For page backgrounds
+
+Examples of hierarchical naming conventions include (from darkest to lightest): 
+1. Strong
+2. Primary
+3. Faint
+4. Disabled (not to be used outside of context)
+
+### Foreground colors
+
+Foreground colors are used for elements such as text, links, statuses, and icons.
+
+Some common examples of semantic foreground colors include:
+- `Foreground / Strong` for headings and secondary links
+- `Foreground / Primary` for body text 
+- `Foreground / Faint` for less prominent text and UI elements.
+- `Foreground / Action` for primary call to actions such as links
+
+![Image showing different text based components with their color tokens associated to them](/assets/foundations/color/colors-foreground-examples.png)
+
+Use status foreground colors to help contextualize responses from user actions or to indicate status within a UI. These colors should be used sparingly and within the context of specific components like [Alerts](/components/alert#color), [Toasts](/components/toast#color) or [Badges](/components/badge#color).
+
+![Image showing different badges with their associated foreground colors](/assets/foundations/color/colors-status-examples.png)
+
+### Border colors
+
+Use border colors for borders on components, containers or dividers.
+
+Some common examples of border colors include:
+- `Border / Strong` for secondary button border
+- `Border / Primary` for Card border or divider
+- `Border / {Status color}` for Alert borders
+
+![Image showing different borders applied on a button, card and alert](/assets/foundations/color/colors-border-examples.png)
+
+### Surface colors
+
+Use surface colors for the background (or surface) of a component or container. 
+
+Some common examples of surface colors include:
+- `Surface / Strong` for the neutral badge.
+- `Surface / Primary` for component containers.
+- `Surface / Faint` for the secondary button.
+- `Surface / {Status color}` for Alert background.
+
+![Image showing different surface colors being applied to a badge, card, button and alert](/assets/foundations/color/colors-surface-examples.png)
+
+### Page colors
+
+Page colors are used for page backgrounds. Our components do not use these tokens, however, we recommend `Page / Primary` as the primary background color and `Page / Faint` as a means to create a secondary level on the page for de-emphasized information, if necessary.
+
+### Disabled colors
```

Comment:
Hmmmm maybe we should remove this section entirely? Especially because we do not advocate for consumers to do this, and because we don't describe other roles, this could be a more appropriate path forward.

Thoughts?

Reply:
Especially because we say:

> Disabled (not to be used outside of context)